src/convertToObject.js

```diff
@@ -2,17 +2,28 @@
 
 /**
  * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const converted = {};
+  const sourceArray = sourceString
+    .split(/\r?\n/)
+    .join('')
+    .split(';')
+    .filter((a) => a.length > 4);
```

Comment:
This is just too much manipulations and each of them comes at its own cost.
Steps:
1. Just split by `;`
2. Do the rest inside the `for loop` (split by `:`, trim and update the `converted` object)